ethToElrond/v2/ethToElrond/steps/step06PerformActionID.go

```diff
@@ -0,0 +1,51 @@
+package steps
+
+import (
+	"context"
+
+	"github.com/ElrondNetwork/elrond-eth-bridge/core"
+	"github.com/ElrondNetwork/elrond-eth-bridge/ethToElrond/v2/ethToElrond"
+)
+
+type performActionIDStep struct {
```

Comment:
can we rename this step? perform action ID doesn't tell much. maybe rename to execute action ID ?

Reply:
the endpoint on SC is performAction